### ISSUE TITLE
kirimase init hangs at Prisma configuration step (v0.0.44) 

### ISSUE THREAD

**Issue by keegn** [nicoalbanese/kirimase#119]
**Config File**
<img width="693" alt="Screenshot 2024-01-18 at 5 21 41 PM" src="https://github.com/nicoalbanese/kirimase/assets/7526703/cf5f9d7c-23e6-47af-b6a4-ff1855fa81ed">
```
{
  "hasSrc": false,
  "packages": [
    "shadcn-ui"
  ],
  "preferredPackageManager": "pnpm",
  "t3": false,
  "alias": "@",
  "rootPath": "",
  "componentLib": "shadcn-ui"
}
```

**Describe the bug**
The DB question (Are you using PlanetScale?) no longer exists after selecting Prisma as the ORM using Kirimase v0.0.44. No issues initializing w/ Prisma using v0.0.43.

After selecting Prisma and misc. packages and hitting enter, it hangs at `Configuring Prisma`.

**To Reproduce**
1. kirimase init
2. select Prisma as ORM
3. hit enter after last init step
4. See screenshot

**Expected behavior**
Successful init

**Comment by nicoalbanese** [comment]
Hey! thanks for flagging - yes silly mistake on my end. Have patched and published a beta version, can you give it a try? 

`bunx kirimase@0.0.45-beta.0 <your-command>`

**Comment by keegn** [comment]
Thanks for the quick fix 🙌! kirimase@0.0.45-beta.0 resolves this

**Comment by nicoalbanese** [comment]
sweet! and no probs, will push it live 😊
